### PR TITLE
Added new security questions about steganograpy and RSA

### PR DESCRIPTION
**Pull request by vivian-dai**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Addresses #104

### CODE REVIEW

File: src/data/security-quiz.ts [hunk 1]
```diff
@@ -266,6 +266,26 @@ const securityQuiz = [
...
+    Link: "https://www.freecodecamp.org/news/steganography-explained-in-plain-english/",
+  },
+  {
+    Question: "What does RSA stand for?",
+    Answer: "Rivest Shamir Adleman",
+    Distractor1: "Really Secure Algorithm",
+    Distractor2: "Router Safe Agreement",
+    Distractor3: "Right Serious Answer",
+    Explanation:
+      "RSA, named after its creators (Rivest, Shamir, Adleman) is an encryption technique that takes advantage of the fact that factoring large prime numbers is hard.",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:286]:
```suggestion
      "RSA, named after its creators (Rivest, Shamir, and Adleman), is an encryption technique that takes advantage of the fact that factoring large prime numbers is hard.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by vivian-dai** [comment]
I've implemented the changes. Let me know if there's any other changes you'd like me to make!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**